test/jdk/javax/swing/JFileChooser/JFileChooserSetLocationTest.java

```diff
@@ -175,12 +180,16 @@ private static Point getActualLocation(final MyFileChooser fcoo)
         return pt.get();
     }
 
-    public static void verify(int x1, int x2, int y1, int y2) {
+    public static void verify(int x1, int x2, int y1, int y2) throws Exception {
         System.out.println("verify " + x1 + "==" + x2 + "; " + y1 + "==" + y2);
         if ((Math.abs(x1 - x2) < TOLERANCE_LEVEL) &&
             (Math.abs(y1 - y2) < TOLERANCE_LEVEL)) {
             System.out.println("Test passed");
         } else {
+            Dimension screenSize = Toolkit.getDefaultToolkit().getScreenSize();
```

Comment:
I think what you want to use here is the bounds of the screen, not the size, since the screen coordinates may start from non-zero values.

Reply:
```
GraphicsConfiguration gc = GraphicsEnvironment.
                         getLocalGraphicsEnvironment().getDefaultScreenDevice().getDefaultConfiguration();
                Rectangle gcBounds = gc.getBounds();
```
This way?

Reply:
yes.